pkg/solana/logpoller/loader.go

```diff
@@ -214,7 +200,7 @@ func (c *EncodedLogCollector) runBlockProcessing(ctx context.Context) {
 				parser:     c.parser,
 				chJobs:     c.chJobs,
 			}); err != nil {
-				c.lggr.Infof("failed to add job to queue: %s", err)
+				c.lggr.Errorf("failed to add job to queue: %s", err)
```

Comment:
These %s's could be changed to %w's I think that's the preferred way to log errors